homeassistant/components/netatmo.py

```diff
@@ -51,13 +53,13 @@ def setup(hass, config):
         NETATMO_AUTH = lnetatmo.ClientAuth(
             config[DOMAIN][CONF_API_KEY], config[DOMAIN][CONF_SECRET_KEY],
             config[DOMAIN][CONF_USERNAME], config[DOMAIN][CONF_PASSWORD],
-            'read_station read_camera access_camera')
+            'read_station read_camera access_camera read_thermostat write_thermostat')
```

Comment:
you have to put a space after 'access_camera' or before 'read_thermostat' otherwise the string will be seen as 'access_cameraread_thermostat'